tensorflow/src/TensorFlow/BuildOp.hs

```diff
@@ -13,6 +13,9 @@
 -- limitations under the License.
 
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures #-}
```

Comment:
I'm suspicious whether any of these added extensions are actually needed for this change in this file :)